src/collection.js

```diff
@@ -104,8 +106,14 @@ export default class Collection {
     });
   }
 
-  use(transfomer) {
-    this._transformers[transfomer.type].push(transfomer);
+  use(plugin) {
```

Comment:
Oh, sorry I misunderstood this! We have different kinds of plugins (adapters, transformers, etc). Ignore these comments.